frontend/src/scenes/session-recordings/player/inspector/playerInspectorLogic.ts

```diff
@@ -284,6 +290,39 @@ export const playerInspectorLogic = kea<playerInspectorLogicType>([
             },
         ],
 
+        browserVisibilityChanges: [
+            (s) => [s.start, s.sessionPlayerData],
+            (start, sessionPlayerData): InspectorListBrowserVisibility[] => {
+                const logs: InspectorListBrowserVisibility[] = []
+
+                Object.entries(sessionPlayerData.snapshotsByWindowId).forEach(([windowId, snapshots]) => {
+                    snapshots.forEach((snapshot: eventWithTime) => {
+                        if (
+                            snapshot.type === 5 // RRWeb custom event type
+                        ) {
+                            const customEvent = snapshot as customEvent
+                            const tag = customEvent.data.tag
+
+                            if (['window hidden', 'window visible'].includes(tag)) {
+                                const { timestamp, timeInRecording } = timeRelativeToStart(snapshot, start)
+                                logs.push({
+                                    type: 'browser-visibility',
+                                    status: tag === 'window hidden' ? 'hidden' : 'visible',
+                                    timestamp: timestamp,
+                                    timeInRecording: timeInRecording,
+                                    search: tag,
+                                    windowId: windowId,
+                                    highlightColor: 'warning',
+                                } satisfies InspectorListBrowserVisibility)
```

Comment:
Should we need to cast here?
```suggestion
                                } as InspectorListBrowserVisibility)
```

Reply:
I think satisfies is better here https://www.totaltypescript.com/clarifying-the-satisfies-operator

(although correct me if I'm misunderstanding :))